app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java

```diff
@@ -147,7 +148,14 @@ public void onNext(Page<Client> clientPage) {
                         getMvpView().hideProgress();
                         if (clientPage.getPageItems().size() != 0) {
                             long clientId = clientPage.getPageItems().get(0).getId();
+                            Context c = context.getApplicationContext();
+                            int mode = Context.MODE_PRIVATE;
+                            String spName = Constants.SHAREDPREF_NAME;
+                            SharedPreferences sp = c.getSharedPreferences(spName, mode);
                             getMvpView().showClient(clientId);
+                            String keyClientId = Constants.SP_KEY_CLIENTID;
+                            sp.edit().putLong(keyClientId, clientId);
+                            sp.edit().commit();
```

Comment:
@shivamarora96 Please use utils/PreManager.class. don't need to make SharedPreferences object.